html-templates/simple.mustache

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
```

Comment:
Simple template to output the data in a table. I will file 3pps for all of the new libs.